File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ChatEventQuery.java
package ru.iqchannels.sdk.schema;


import androidx.annotation.Nullable;

/**
 * Copyright iqstore.ru. All Rights Reserved.
 */
public class ChatEventQuery {
    @Nullable public Long LastEventId;
    @Nullable public Integer Limit;
    
    public ChatEventQuery() {}
    
    public ChatEventQuery(@Nullable Long lastEventId) {
        this.LastEventId = lastEventId;
    }
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ChatEventQuery.kt
package ru.iqchannels.sdk.schema

class ChatEventQuery {
	var LastEventId: Long? = null
	var Limit: Int? = null
	var ChatType: String = ru.iqchannels.sdk.domain.models.ChatType.REGULAR.name.lowercase()

	constructor()
	constructor(lastEventId: Long?) {
		LastEventId = lastEventId
	}
}
